setup-etcd/action.yml

```diff
@@ -18,7 +18,15 @@ inputs:
 runs:
   using: composite
   steps:
+    - name: Cache etcd binaries
+      id: cache-etcd-binaries
+      uses: actions/cache@v4
+      with:
+        path: ${{ inputs.install-prefix }}
+        key: etcd-binaries-${{ runner.os }}-${{ inputs.version }}-${{ inputs.platform }}-${{ inputs.install-prefix }}-v1
```

Comment:
1. Why `v1` at the end?
2. Is `${{ inputs.install-prefix }}` really needed in the key?

Reply:
1. It was the usual workaround of https://github.com/actions/cache/issues/2 in workflow files. The workaround is not needed anymore. However, here we have an action, not a workflow. We can't clear a cache before a commit that changes the caching logic: we don't control lifecycles of the cached files. So, it seems logical to use the same approach to discard old caches as we have been used for workflow files in the past. If we need to change the caching logic, we should change v1 to v2.
2. We can split the step to several steps and cache just downloaded archive: in this case we shouldn't. In the given case, I'm not sure. The step itself depends on this input, so it seems that the cache key should depend on it. I looked briefly, but I failed to find a detailed description how the collection/extraction works, so I can't say for sure.

Reply:
OK, thanks!